content/docs/04.5-devx/07-sandbox-clusters.md

```diff
@@ -47,3 +47,57 @@ To create your new sandbox cluster:
    * Review the information and deploy the Palette virtual cluster. The Palette virtual cluster will be provisioned within the next few minutes.
 
 <br />
+
+
+## Palette Virtual Cluster Pause and Resume
+
+The Palette allows the Pause and Resume of Palette Virtual Clusters when not in use. This feature enables the users with optimized resource sharing by pausing the virtual clusters, not in use. This adds significant flexibility in managing operating costs and resource management for the Palette virtual clusters. 
```

Comment:
The Palette ??? Did you mean the Palette platform?

Reply:
Pause and Resume are not nouns, so they should not be in title case.

Reply:
Is it only for users with optimized resources? It's a feature for all users that helps them manage resources more efficiently but more importantly, better manage the cost of running clusters.

Reply:
Palette Virtual Clusters should be in title case.